Graphics/Implicit/Primitives.hs

```diff
@@ -5,6 +5,7 @@
 -- FIXME: Required. why?
 {-# LANGUAGE FunctionalDependencies #-}
 {-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE CPP #-}
```

Comment:
Eew?

Reply:
I try to avoid CPP, but it has its purposes, and generation of identical instances is one of them. This pattern is somewhat common in cases like these, cf:

* https://hackage.haskell.org/package/vector-space-0.16/docs/src/Data.VectorSpace.html#project
* http://hackage.haskell.org/package/monad-control-1.0.2.3/docs/src/Control-Monad-Trans-Control.html#RunInBase